interpreter/CppInterOp/unittests/CppInterOp/FunctionReflectionTest.cpp

```diff
@@ -868,6 +868,10 @@ TEST(FunctionReflectionTest, JitCallAdvanced) {
 TEST(FunctionReflectionTest, GetFunctionCallWrapper) {
   if (llvm::sys::RunningOnValgrind())
     GTEST_SKIP() << "XFAIL due to Valgrind report";
+
+#ifdef _WIN32
+  GTEST_SKIP() << "Disabled on Windows (Fails with ROOT win-x64)";
+#endif
```

Comment:
Is this done upstream in CppInterOp? We should not start by immediately diverging

Reply:
@aaronj0, do you remember how this used to fail?